### ISSUE TITLE
Google and Bing not working

### ISSUE THREAD

**Issue by henkieaardpeer** [uyjulian/Translation-Aggregator#12]
Hello, 

there were only two online translation engines left that were still working for me: Google and Bing. But now neither of them works and I can't play any game.

Textractor seems to be having the same problem and it also seems to have been fixed today in a new release: 
https://github.com/Artikash/Textractor/issues/177 (Bing issue)
https://github.com/Artikash/Textractor/issues/172 (Google issue)

### So, to summarize: TA needs an update because things have changed.

(I also used LEC Online in the past, but it seemed to have permanently banned me due to more than 2 million requests or something like that.)

![afbeelding](https://user-images.githubusercontent.com/40700989/61580366-9428b400-ab11-11e9-9e69-b553f0b1b727.png)

**Comment by uyjulian** [comment]
v1.0.2 is now available: https://github.com/uyjulian/Translation-Aggregator/releases/tag/v1.0.2
